package.json

```diff
@@ -18,12 +18,12 @@
     "docs": "typedoc",
     "lint": "eslint --ignore-path .gitignore \"src/**/*.ts\" --fix",
     "prepublishOnly": "npm run clean:all && npm i && npm run build && npm t",
-    "start:e2e": "ws -d tests/e2e/app  -p 8443 --cors.origin null --cors.credentials --cert ./example.com+5.pem --key ./example.com+5-key.pem",
```

Comment:
Because we remove the certificates here, and use the ones in `lws` we can leverage the `--https` option.